dist/index.html

```diff
@@ -4,9 +4,57 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="index.css">
   <title>Document</title>
 <script defer src="main.js"></script></head>
 <body>
+  <header>
+    <nav class="container">
+      <h1 class="header">Awesome Books</h1>
+      <ul class="nav">
+        <li class="menu" id="list">List</li>
+       
+        <li class="menu" id="add">Add</li>
+     
+        <li class="menu" id="contact">Contact</li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class="time">
+    <p class="date" id="time"></p>
+  </div>
   
+  <main>
+    <section class="booklist">
+    
+    </section>
```

Comment:
- [ ] After solving the bug that is discussed above. I found out that you tried updating the design in this section. Well done 👏🏼. However, there is still some improvement that is expected here.

| Your Implementation | The Wireframe |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/45002654/236237287-f5165268-bb0d-4fa9-9202-fddbd0e04874.png) | ![image](https://user-images.githubusercontent.com/45002654/235494033-7ceb4f83-bf12-449d-9e6d-081761a78650.png) |

Please do not ignore the following
- Alternating backgrounds
- Borders
- Styling for the delete button
- The book title should be in quotes
- There should be the word "by" in between the Title and the Author